doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,351 @@
+different host port (``50053``) to avoid port conflicts with local
+MAPDL instances running on the host or other Docker images.
+You could additionally launch more Docker containers in different ports if
+you want to run multiple simulations at the same time.
+
+The :ref:`ref_pymapdl_pool`` module does not work when you are connecting
+to a remote MAPDL Docker image. It does not work when connected to Docker
+containers either. If you decide to launch multiple MAPDL instances, you must
+manage these instances yourself.
+
+.. note:: Ensure that port ``50053`` is open in your local firewall.
+
+You can provide additional MAPDL command line parameters to MAPDL by simply
+appending them to the end of the command.
+
+For example, you can increase the number of processors (up to the
+number available on the host machine) with the ``-np`` switch:
+
+.. code:: bash
+
+  docker run -e ANSYSLMD_LICENSE_FILE=$ANSYSLMD_LICENSE_FILE --restart always -d --name mapdl -p $LOCAL_MAPDL_PORT:50052 $MAPDL_DOCKER_REGISTRY_URL -smp -np 8 > log.txt
+
+
+For additional command line arguments, see the *Notes* section in the
+description for the :func:`launch_mapdl() <ansys.mapdl.core.launch_mapdl>`
+function.
+
+You can use a script file (batch ``'.bat'`` or PowerShell ``'.ps'``)
+to run the preceding commands all at once.
+
+Once you have launched MAPDL, you should see the following content
+in your console (or the output file):
+
+.. code::
+
+    Start GRPC Server
+
+    ##############################
+    ### START GRPC SERVER      ###
+    ##############################
+
+    Server Executable   : MapdlGrpc Server
+    Server listening on : 0.0.0.0:50052
+
+
+.. warning:: 
+  
+   Notice that the port specified in the console is the internal Docker container port.
+   This port has been mapped to the value specified for the :envvar:`LOCAL_MAPDL_PORT`
+   environment variable.
+
+
+.. _run_an_mapdl_image_using_docker_compose:
+
+Using ``docker-compose`` to launch MAPDL
+----------------------------------------
+
+You can also use `docker-compose <docker_compose_>`_ command to launch MAPDL configured in
+a `docker-compose <docker_compose_>`_ file.
+This is useful if you want to load an already configured environment, or
+if you want to launch multiple instances of MAPDL or services.
+
+For you convenience, the directory `docker <pymapdl_docker_dir_>`_ 
+contains pre-configured `docker-compose <docker_compose_>`_ files that you can
+use.
+It is recommended to use this `docker-compose.yml <pymapdl_docker_compose_base_>`_ file.
+This is the **base** configuration file which launches an instance of MAPDL which you
+can connect to as remote.
+
+
+.. _pymapdl_connect_to_MAPDL_container:
+
+Connect to the MAPDL container from Python
+==========================================
+
+You can connect to the MAPDL instance following :ref:`connect_grpc_madpl_session`.
+You do not need to specify an IP since Docker maps the ports to the local host.
```

Comment:
```suggestion
You can connect to an MAPDL instance as indicated in :ref:`connect_grpc_madpl_session`.
You do not need to specify an IP address because Docker maps the ports to the local host.
```